### PR TITLE
phungminhthi/feature/t04-search-job-by-title

### PR DESCRIPTION
**Pull request by minhthiphung989**
-  Use filter() and use If  else to compare the value with the jobName of the data.
-  Pagecount of pagination will change when search jobs.
- Link deploy: https://flawless-rest.surge.sh/

### CODE REVIEW

File: src/action/actions.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (nhatphfaba) [inline src/action/actions.tsx:1]: please finish rebase on this file

File: src/components/AllJobs/AllJobs.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React, { FC, useEffect } from 'react'
 import './_AllJobs.scss'
+<<<<<<< HEAD
```
Comment (nhatphfaba) [inline src/components/AllJobs/AllJobs.tsx:3]: please finish rebase on this file

File: src/components/AllJobs/_AllJobs.scss [hunk 3]
```diff
@@ -42,4 +42,8 @@
         }
     }
     
-}
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (nhatphfaba) [inline src/components/AllJobs/_AllJobs.scss:45]: please check rebase issues

**Review by nhatphfaba — CHANGES_REQUESTED** [review]

**Comment by nhatphfaba** [comment]
Please check again all conflic issues before submiting code.

File: src/action/actions.tsx (hunk 1 above)
Comment (minhthiphung989) [reply src/action/actions.tsx]: fixed

File: src/components/AllJobs/AllJobs.tsx (hunk 2 above)
Comment (minhthiphung989) [reply src/components/AllJobs/AllJobs.tsx]: fixed

File: src/components/AllJobs/_AllJobs.scss (hunk 3 above)
Comment (minhthiphung989) [reply src/components/AllJobs/_AllJobs.scss]: fixed

**Pull request closed without merging**